### PR TITLE
Updated The firebaseui due to 3rd-party cookies

### PR DESCRIPTION
**Pull request by mabels**
Here is my updated firebaseui patch which is based on my config update which you rejected out of the reason it don't work. I requested some help from you:)

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -1,14 +1,93 @@
...
+if (isLocal && typeof local.postgresSsl === 'boolean') {
+    ssl = local.postgresSsl;
+} else if (typeof process.env.POSTGRES_SSL === 'string') {
+    const sslString = process.env.POSTGRES_SSL;
+    if (sslString.startsWith('F') || sslString.startsWith('f') || parseInt(sslString, 10) === 0) {
+        ssl = false;
+    }
+}
+export const postgres = {
+    connectionString: isLocal ? local.postgresConnectionString : process.env.POSTGRES_CONNECTIONSTRING,
```
Comment (andrei-tatar) [inline src/config.ts]: I think the problem comes from then env variables. Can you leave them with the name they had?

Comment (mabels) [reply src/config.ts]: Sorry for that, I had a bit lazy on the last changes. I now made these three env-vars:
DATABASE_URL
SERVICE_ACCOUNT_ISSUER
SERVICE_ACCOUNT_KEY
backward compatible. Hopefully I get all of the non compatible changes yet.

Comment (mabels) [reply src/config.ts]: I might be an good idea to transform the configure to an service. But on the other hand I need to make the @Get('/whatever') annotations working with an dynamic injection an Base director. This is somehow needed for the google cloud function runtime. What do you thing might be the best why. To make that dynamic this function like:
@Get('/whatever', cfg.baseDir) or to inject the config into the AnnotationService.

Comment (andrei-tatar) [reply src/config.ts]: Not sure I understand the context. Why do you need the config in `@get`?

Comment (mabels) [reply src/config.ts]: Basically I  will change the currently static config to an injected object.
To run on firebase the url locals like: 
https://firebase/base/.....
So your pattern for adding a route has do be configure dependable. Thats why I thought to added an parameter to cfg. 
So just don't care now, I will introduce next the configuration to be injectable.

Comment (andrei-tatar) [reply src/config.ts]: If you want to add a prefix (like `base` or `api`) to all the routes you can do it directly in `src/http/app.ts` https://github.com/andrei-tatar/nora-service/blob/master/src/http/app.ts#L21

Comment (mabels) [reply src/config.ts]: we need that information also in the redirects 302. But currently I'm fine with that.

**Pull request merged**